db/models/__init__.py

```diff
@@ -40,54 +41,55 @@ def dict(self, **kwargs) -> dict:
 class FlowCheckDoc(DocumentBaseModel):
     """FlowCheckDoc."""
 
-    state = "active"
+    state: str = Field(default="active")
```

Comment:
Looks like this can be just `state: str = "active"`, doesn't it?

Reply:
Yes, changed